src/main/java/com/ceritoxi/util/Math.java

```diff
@@ -0,0 +1,470 @@
+package com.ceritoxi.util;
+import java.util.Stack;
+import static java.lang.Math.sqrt;
+
+public class Math {
+
+    public static boolean isDividableBy(int dividend, int number) {
+        return (number != 0 && dividend % number == 0);
+    }
+
+    public static int sumOfNumbersUnderZThatAreDividableByXorY (int x, int y, int z) {
+        int sum = 0;
+
+        for (int i = 1; i < z; i++) {
+            if (isDividableBy(i, x) || isDividableBy(i, y)) {
+                sum += i;
+            }
+        }
+        return sum;
+    }
+
+    public static boolean isEven (int number) {
+        return number % 2 == 0;
+    }
+
+    public static boolean isEven (long number) {
+        return number % 2 == 0;
```

Comment:
2 and 0 is still an int so there will be a conversion one way or the other. If you really want a separate isEven function those should be declared with 2L and 0L

Reply:
Yep I forgot about that one.